src/App.js

```diff
@@ -10,7 +10,8 @@ import './layouts/Gallery.css';
 function App(props) {
     console.log(props);
     return (
-        <div className="Gallery">
+        // className="Gallery" probably unintentional, "Gallery" is already being used below
```

Comment:
oops, yea i was messing around with a lot of stuff and tried to get it to format